translations/translations.json

```diff
@@ -3979,6 +3979,7 @@
       "changeLanguage": "Sprache wechseln",
       "settings": "Einstellungen",
       "copyUrl": "Link kopieren",
+      "feedback": "Feedback",
```

Comment:
Where do we need this?

Reply:
FeedbackToolbarItem LN44

Reply:
Actually not, we use namespace `feedback` there. However, I think you just readded it after removing it accidentally, right? So just keep it.